apps/token-e2e/src/support/common.functions.js

```diff
@@ -14,13 +14,23 @@ const navigation = {
   withdraw: '[href="/token/withdraw"]',
   proposals: '[href="/proposals"]',
   pageSpinner: '[data-testid="splash-loader"]',
+  supply: '[href="/token/tranches"]',
   token: '[href="/token"]',
 };
 
 Cypress.Commands.add('navigate_to', (page) => {
-  return cy.get(navigation.section, { timeout: 10000 }).within(() => {
-    cy.get(navigation[page]).click({ force: true });
-  });
+  const tokenDropDown = 'state-trigger';
+
+  if (page != 'proposals' && page != 'validators' && page != 'rewards') {
```

Comment:
Suggestion: I'm not a big fan of these routes being hard coded here. I've dropped a comment below about separating out the page structure from the subnav element, that config could then be used here to build this conditional